service/debugger/debugger.go

```diff
@@ -39,6 +39,8 @@ type Debugger struct {
 	target       proc.Process
 	log          *logrus.Entry
 
+	canRestart bool // target was started with Launch and can be restarted
```

Comment:
Can't we infer this from `config` by asserting that we weren't given a PID to attach to?